Sources/SwiftCurrent_SwiftUI/WorkflowItem.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  WorkflowItem.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Tyler Thompson on 7/12/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+import SwiftCurrent
+
+/**
+ A concrete type used to modify a `FlowRepresentable` in a `WorkflowView`.
+
+ ### Discussion
+ `WorkflowItem` gives you the ability to specify changes you'd like to apply to a specific `FlowRepresentable` when it is time to present it in a `Workflow`.
+
+ #### Example
+ ```swift
+ WorkflowItem(FirstView.self)
+            .persistence(.removedAfterProceeding) // affects only FirstView
+            .applyModifiers {
+                if true { // Enabling transition animation
+                    $0.background(Color.gray) // $0 is a FirstView instance
+                        .transition(.slide)
+                        .animation(.spring())
+                }
+            }
+ ```
+ */
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+public final class WorkflowItem<F: FlowRepresentable & View> {
+    var metadata: FlowRepresentableMetadata!
+    private var flowPersistenceClosure: (AnyWorkflow.PassedArgs) -> FlowPersistence = { _ in .default }
+    private var modifierClosure: ((AnyFlowRepresentableView) -> Void)?
+
+    /// Creates a `WorkflowItem` with no arguments from a `FlowRepresentable` that is also a View.
+    public init(_: F.Type) {
+        metadata = FlowRepresentableMetadata(F.self,
+                                             launchStyle: .new,
+                                             flowPersistence: flowPersistenceClosure,
+                                             flowRepresentableFactory: factory)
+    }
+
+    /**
+     Provides a way to apply modifiers to your `FlowRepresentable` view.
+
+     ### Important: The most recently defined (or last) use of this, is the only one that applies modifiers, unlike onAbandon or onFinish.
+     */
+    public func applyModifiers<V: View>(@ViewBuilder _ closure: @escaping (F) -> V) -> Self {
+        modifierClosure = {
+            // We are essentially casting this to itself, that cannot fail. (Famous last words)
+            // swiftlint:disable:next force_cast
+            let instance = $0.underlyingInstance as! F
+            $0.changeUnderlyingView(to: closure(instance))
+        }
+        return self
+    }
+
+    func factory(args: AnyWorkflow.PassedArgs) -> AnyFlowRepresentable {
```

Comment:
could this be private?
```suggestion
    private func factory(args: AnyWorkflow.PassedArgs) -> AnyFlowRepresentable {
```

Reply:
made private